hikari/internal/enums.py

```diff
@@ -35,14 +35,45 @@
 _MAX_CACHED_MEMBERS: typing.Final[int] = 1 << 12
 
 
+class _DeprecatedAlias(typing.Generic[_T]):
+    __slots__ = ("_name", "_alias", "_removal_version")
+
+    def __init__(self, name: str, other: _T, removal_version: str) -> None:
+        self._name = name
+        self._alias = other
+        self._removal_version = removal_version
+
+    def __get__(self, instance: typing.Optional[_T], owner_class: _T) -> _T:
+        # Import kept in-line because of circular import issues
+        from hikari.internal import deprecation
+
+        deprecation.warn_deprecated(
+            self._name,
+            removal_version=self._removal_version,
+            additional_info=f"Use '{self._alias}' instead.",
+        )
+
+        return self._alias
```

Comment:
this is returning the enum member's name rather than value which seems like not what you wanted